package.json

```diff
@@ -65,8 +65,7 @@
     "onCommand:pacCLI.openDocumentation",
     "onCommand:microsoft-powerapps-portals.webExtension.init",
     "onStartupFinished",
```

Comment:
If you have `onStartupFinished` you will activate for every window of vscode. I recommend removing this one.

Reply:
We need to register values in the PATH environment variable prior to a Terminal opening, so we cannot remove `onStartupFinished` unless there is more specific way to target that scenario.

Reply:
Unless the terminal is interacted with it should be relaunched automatically. Plus I believe * activation can happen before the first terminal launches anyway. Using * slows down all vscode launches, even when you're not using the extension.